src/App.js

```diff
@@ -1,11 +1,11 @@
 import React, {Component} from "react";
-import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
 
 import logo from "./logo.svg";
 import "./App.css";
 
-import ChatWindow from "./ChatWindow.js";
-import LoginControl from "./LoginControl.js";
+import Header from "./Header.js";
```

Comment:
.js is not needed I think. Check and if it is not needed, remove it across all of your imports.

Reply:
Removed .js across all files.